gossip/src/restart_crds_values.rs

```diff
@@ -122,6 +133,45 @@ impl RestartLastVotedForkSlots {
     }
 }
 
+impl Sanitize for RestartHeaviestFork {
+    fn sanitize(&self) -> Result<(), SanitizeError> {
+        sanitize_wallclock(self.wallclock)?;
+        self.last_slot_hash.sanitize()
+    }
+}
+
+impl RestartHeaviestFork {
+    pub fn new(
+        from: Pubkey,
+        now: u64,
+        last_slot: Slot,
+        last_slot_hash: Hash,
+        observed_stake: u64,
+        shred_version: u16,
+    ) -> Self {
+        Self {
+            from,
+            wallclock: now,
+            last_slot,
+            last_slot_hash,
+            observed_stake,
+            shred_version,
+        }
+    }
```

Comment:
Why do we need this? all fields are public.
Either remove fields from public interface or remove this redundant method.

Reply:
Removed.